platform-configs/include/pcengines.robot

```diff
@@ -0,0 +1,115 @@
+${DEF_SOCKETS}=                                     1
+${POWER_CTRL}=                                      RteCtrl
+${FLASH_VERIFY_METHOD}=                             tianocore-shell
+${FLASH_VERIFY_OPTION}=                             UEFI Shell
+# TODO
+${MAX_CPU_TEMP}=                                    ${EMPTY}
+
+${DMIDECODE_MANUFACTURER}=                          PC Engines
+${DMIDECODE_VENDOR}=                                3mdeb
+${DMIDECODE_FAMILY}=                                N/A
+# TODO
+${DMIDECODE_TYPE}=                                  Desktop
+
+# Supported test environments
+${TESTS_IN_FIRMWARE_SUPPORT}=                       ${TRUE}
+${TESTS_IN_UBUNTU_SUPPORT}=                         ${TRUE}
+
+# Regression test flags
+${DASHARO_SECURITY_MENU_SUPPORT}=                   ${TRUE}
+# Test module: dasharo-compatibility
+${CUSTOM_BOOT_MENU_KEY_SUPPORT}=                    ${TRUE}
+${CUSTOM_SETUP_MENU_KEY_SUPPORT}=                   ${TRUE}
+${EC_AND_SUPER_IO_SUPPORT}=                         ${TRUE}
+${USB_DISKS_DETECTION_SUPPORT}=                     ${TRUE}
+${UEFI_SHELL_SUPPORT}=                              ${TRUE}
+${UEFI_COMPATIBLE_INTERFACE_SUPPORT}=               ${TRUE}
+${IPXE_BOOT_SUPPORT}=                               ${TRUE}
+${SD_CARD_READER_SUPPORT}=                          ${TRUE}
+${WIRELESS_CARD_SUPPORT}=                           ${TRUE}
+${WIRELESS_CARD_WIFI_SUPPORT}=                      ${TRUE}
+${SERIAL_NUMBER_VERIFICATION}=                      ${TRUE}
+${FIRMWARE_NUMBER_VERIFICATION}=                    ${TRUE}
+${FIRMWARE_FROM_BINARY}=                            ${TRUE}
+${PRODUCT_NAME_VERIFICATION}=                       ${TRUE}
+${RELEASE_DATE_VERIFICATION}=                       ${TRUE}
+${RELEASE_DATE_FROM_SOL}=                           ${TRUE}
+${MANUFACTURER_VERIFICATION}=                       ${TRUE}
+${VENDOR_VERIFICATION}=                             ${TRUE}
+${FAMILY_VERIFICATION}=                             ${TRUE}
+${TYPE_VERIFICATION}=                               ${TRUE}
+${HARDWARE_WP_SUPPORT}=                             ${TRUE}
+${DTS_SUPPORT}=                                     ${TRUE}
+${FIRMWARE_BUILDING_SUPPORT}=                       ${TRUE}
+${UPLOAD_ON_USB_SUPPORT}=                           ${TRUE}
+${CPU_TESTS_SUPPORT}=                               ${TRUE}
+${RESET_TO_DEFAULTS_SUPPORT}=                       ${TRUE}
+${L2_CACHE_SUPPORT}=                                ${TRUE}
+${L3_CACHE_SUPPORT}=                                ${TRUE}
+${DEFAULT_POWER_STATE_AFTER_FAIL}=                  Powered On
+${ESP_SCANNING_SUPPORT}=                            ${TRUE}
+${DTS_FIRMWARE_FLASHING_SUPPORT}=                   ${TRUE}
+${BASE_PORT_BOOTBLOCK_SUPPORT}=                     ${TRUE}
+${BASE_PORT_ROMSTAGE_SUPPORT}=                      ${TRUE}
+${BASE_PORT_POSTCAR_SUPPORT}=                       ${TRUE}
+${BASE_PORT_RAMSTAGE_SUPPORT}=                      ${TRUE}
```

Comment:
These are only for Talos IIRC

Reply:
Ack, removed

Reply:
https://github.com/Dasharo/open-source-firmware-validation/pull/211/commits/9e3ed55a6642f346a9b2a0e0204655eb0acf407e